CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -211,15 +213,31 @@ private void AnalyseObjectForBinding(JavascriptObject obj, bool analyseMethods,
                     var jsObject = CreateJavascriptObject(camelCaseJavascriptNames);
                     jsObject.Name = propertyInfo.Name;
                     jsObject.JavascriptName = GetJavascriptName(propertyInfo.Name, camelCaseJavascriptNames);
-                    jsObject.Value = jsProperty.GetValue(obj.Value);
+                    jsObject.LateBinding = () =>
+                    {
+                        if (!propertyInfo.PropertyType.IsArray)
+                        {
+                            jsObject.Value = jsProperty.GetValue(obj.Value);
+                            AnalyseObjectForBinding(jsObject, camelCaseJavascriptNames);
+                        }
+                        else
+                        {
+                            Array array = jsProperty.GetValue(obj.Value) as Array;
```

Comment:
Perhaps I wasn't clear, regardless if your going to check for null then the `as` is perfect.